hydromt/cli/main.py

```diff
@@ -464,31 +465,58 @@ def export(
     if dd and "deltares_data" not in data_libs:  # deltares_data from cli
         data_libs = ["deltares_data"] + data_libs  # prepend!
 
-    if export_config:
-        args = export_config.pop("args", {})
+    sources: List[str] = []
+
+    if source:
+        if isinstance(source, str):
+            sources = [source]
+        else:
+            sources = list(source)
+
+    if config:
+        config_dict = cli_utils.parse_config(config)["export_data"]
+        args = config_dict.pop("args", {})
         if "catalog" in args.keys():
```

Comment:
I suggested data_libs in the issue instead of catalog in the template (not as a bullet point). I leave up to you to change but data_libs is used in the build and update command to specify catalogs from config file ([global] section of the config corresponding to Model.__init__ keys)